src/clojurians_log/routes.clj

```diff
@@ -156,16 +165,22 @@
 (def routes
   [["/" {:name :clojurians-log.routes/index
          :get index-route}]
+   ["/_/_/users/{user-id}" {:name :clojurians-log.routes/user-profile-route
+                          :get user-profile-route}]
    ["/x/x/x/about" {:name :clojurians-log.routes/about
                     :get about-route}]
    ["/x/x/x/sitemap" {:name :clojurians-log.routes/sitemap
                       :get sitemap-route}]
-   ["/x/x/x/healthcheck" {:name :clojurians-log.routes/healthcheck,
+   ["/x/x/x/healthcheck" {:name :clojurians-log.routes/healthcheck
                           :get healthcheck-route}]
-   ["/{channel}" {:name :clojurians-log.routes/channel,
+   ["/{channel}" {:name :clojurians-log.routes/channel
                   :get channel-history-route}]
-   ["/{channel}/{date}" {:name :clojurians-log.routes/channel-date,
+   ["/{channel}/{date}" {:name :clojurians-log.routes/channel-date
                          :get log-route}]
-   ["/{channel}/{date}/{ts}" {:name :clojurians-log.routes/message,
-                              :get log-route}]
-   ["/_/stats/{from-date}/{to-date}" {:name :clojurians-log.routes/message-stats :get message-stats-route}]])
+   ["/{channel}/{date}/{ts}" {:name :clojurians-log.routes/message :get log-route}]
+   ["/_/stats/{from-date}/{to-date}" {:name :clojurians-log.routes/message-stats 
+                                      :get message-stats-route}]
+
+   {:conflicts nil}
```

Comment:
Is this still needed?

Reply:
No, changing the route for the user profile page to "/_/_/users/{user-id}" fixed this.